### ISSUE TITLE
Submit relay entry to the chain

### ISSUE THREAD

**Issue by pdyraga** [keep-network/keep-core#3046]
Depends on #3045 
Submitting new entry to the chain with `function submitRelayEntry(bytes calldata entry)` or function `submitRelayEntry(bytes calldata entry, uint32[] calldata groupMembers)`, depending on the time left before the hard timeout.

**Comment by mswilkison** [comment]
Consolidated into the random-beacon tracking epic #3044. This isn't independent work — it's blocked on the same missing layer (wiring `pkg/chain/ethereum/beacon.go` to the generated `RandomBeacon` bindings), which is deliberately deferred behind FROST/ROAST GA → a permissionless-participation decision. The current chaosnet stub is fit for the permissioned operator set. Full evidence and the dependency chain are in #3044; reopen there if the roadmap changes.
